### PR TITLE
Fix validation bug on Piped config

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/config/piped.go [hunk 1]
```diff
@@ -77,8 +77,11 @@ func (s *PipedSpec) Validate() error {
 	if s.PipedID == "" {
 		return errors.New("pipedID must be set")
 	}
-	if s.PipedKeyFile == "" {
-		return errors.New("pipedKeyFile must be set")
+	if s.PipedKeyData == "" && s.PipedKeyFile == "" {
+		return errors.New("either pipedKeyFile or pipedKeyData must be set")
+	}
+	if s.PipedKeyData != "" && s.PipedKeyFile != "" {
+		return errors.New("only either pipedKeyFile or pipedKeyData can be set")
```
Comment (khanhtc1202) [inline pkg/config/piped.go:84]:
```suggestion
		return errors.New("only pipedKeyFile or pipedKeyData can be set")
```
nits

File: pkg/config/piped.go [hunk 2]
```diff
@@ -194,9 +197,6 @@ func (s *PipedSpec) GetSecretManagement() *SecretManagement {
 }
 
 func (s *PipedSpec) LoadPipedKey() ([]byte, error) {
-	if s.PipedKeyData != "" && s.PipedKeyFile != "" {
```
Comment (nakabonne) [inline pkg/config/piped.go:197]: Right

**Comment by nakabonne** [comment]
It should 👍 
/lgtm

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**